torch/_inductor/codegen/cuda/cutlass_utils.py

```diff
@@ -89,6 +89,34 @@ def try_import_cutlass() -> bool:
 
         return True
 
+    try:
+        import cutlass  # type: ignore[import-not-found]  # noqa: F811
+        import cutlass_library  # type: ignore[import-not-found]  # noqa: F811
+
+        cutlass_minor_vesion = int(cutlass.__version__.split(".")[1])
```

Comment:
Um isn't the way of checking versions deprecated and we should use importlib metadata?

Reply:
Probably. I just reverted the commit that removed this restoring the original code and didn't want to change it.

Anyway as it's good going forward I did that, fixed the typo and actually check for ` < 3.7` and not fail for e.g. `4.1` when only checking the minor version.